src/components/EventForm/EventForm.js

```diff
@@ -39,149 +40,155 @@ function EventForm({courses, friends}) {
     return checked.map((f) => f.value);
   };
 
-  const submitForm = () => {
-    const event = {
-      golfCourse, date, teeTime, openSpots, numHoles, selectedFriends
-    }
-    console.log(event);
+  const submitForm = (e) => {
+    postEvent(golfCourse, date, teeTime, openSpots, numHoles, isPrivate, hostId, selectedFriends)
+      .then(json => console.log(json))
+      .catch(e => console.log(e))
```

Comment:
At some point, we could set this error to state and render it on the page (if we have time).